workflow-job

```diff
@@ -224,14 +228,38 @@ def isAlive(dockerContainer) {
    }
 }
 
+def waitGuiceServer(dockerContainer, jamesCliCmd, deploying) {
+    echo 'Debug step 1'
+    echo "${dockerContainer}"
+    echo 'Debug step 2'
+    echo "${jamesCliCmd}"
+    echo 'Waiting for James server to be deployed'
+    try {
+        waitForCommandSuccess("docker exec ${dockerContainer} ${jamesCliCmd} listusers", maxRetries)
+        echo 'Debug is done on waiting'
+    } catch (Exception e) {
+        echo 'Debug with problem'
+        sh "docker logs ${dockerContainer}"
+    }
+    echo 'Debug step 3'
+    deploying.success();
+    echo 'Debug step 4'
+}
+
+def runTestOnGuiceServer(dockerContainer, jamesCliCmd) {
```

Comment:
It's not provision in fact, it's just a simple command to make sure that server is launched correctly so I don't think the method name is provision. WDYT?